docs/source/contributing/developer_guide.rst

```diff
@@ -204,8 +204,8 @@ distribution. It has the following signature:
 
 In the ``logp`` method, parameters and values are either Aesara tensors,
 or could be converted to tensors. It is rather convenient as the
-evaluation of logp is represented as a tensor (``RV.logpt``), and when
-we linked different ``logp`` together (e.g., summing all ``RVs.logpt``
+evaluation of logp is represented as a tensor (``RV.logp``), and when
```

Comment:
This whole developer guide is outdated, so let's just leave it unchanged for now